frontend/src/components/CardDetailsPage.tsx

```diff
@@ -213,6 +290,11 @@ const DetailsCard = ({
               <RepositoriesCard maxInitialDisplay={4} repositories={repositories} />
             </SecondaryCard>
           )}
+        {type === 'program' && modules.length > 0 && (
```

Comment:
Let's change this to only show this small cards like this if there's more than 1 module. 
If there's just one - it should be a default one that we can show on the same page.

<img width="1213" height="300" alt="Screenshot 2025-08-06 at 6 20 31 PM" src="https://github.com/user-attachments/assets/d405d524-eefa-40e9-9fc3-9b042a45828a" />

I was thinking we could have a separate view in that case, where the module info take full space in this secondary card. We can extend this to show description, as well as mentors. The `Edit` button would also be shown here in that case (only if it's under `my/mentorship` view, as mentioned above). 

If there are more than one - we'll show it the same way.